driver/src/price_finding/optimization_price_finder.rs

```diff
@@ -284,14 +289,21 @@ impl Io for DefaultIo {
         solver_type: SolverType,
         time_limit: Duration,
     ) -> Result<()> {
-        let time_limit = (time_limit.as_secs_f64().round() as u64).to_string();
-        let output = Command::new("python")
-            .args(&["-m", "batchauctions.scripts.e2e._run"])
-            .arg(input_file)
-            .args(&["--outputDir", result_folder])
-            .args(&["--solverTimeLimit", &time_limit])
-            .args(solver_type.to_args())
-            .output()?;
+        let mut command = {
+            let time_limit = (time_limit.as_secs_f64().round() as u64).to_string();
+            let mut command = Command::new("python");
+            command
+                .args(&["-m", "batchauctions.scripts.e2e._run"])
+                .arg(input_file)
+                .args(&["--outputDir", result_folder])
+                .args(&["--solverTimeLimit", &time_limit])
+                .args(solver_type.to_args());
+
+            debug!("Using solver command `{:?}`", command);
+            command
+        };
+
+        let output = command.output()?;
```

Comment:
Should this get a context too?

Reply:
The `IO::run_solver` has error context already, and if the exit status is non-zero there is an `error!` log message.